google-compute-engine/src/test/java/org/jclouds/googlecomputeengine/features/InstanceApiLiveTest.java

```diff
@@ -165,6 +212,19 @@ public void testGetSerialPortOutput() {
       assertNotNull(output.contents());
    }
 
+   @Test(groups = "live", dependsOnMethods = "testInsertInstanceWindows")
+   public void testGetSerialPortOutput4() throws NoSuchAlgorithmException, CertificateException {
+      Crypto crypto = new BouncyCastleCrypto();
+      Security.addProvider(new org.bouncycastle.jce.provider.BouncyCastleProvider());
+      Instance instance = api().get(INSTANCE_NAME_WINDOWS);
+
+      // TODO instantiate better way ResetWindowsPassword
+      // TODO think about not using it at all
```

Comment:
Agree, in this test you just need to call the API and verify the call itself works. There is no need to call the reset password logic here.
If you want to do so, however, you'dd better override the `create` method of the parent class so you can get a `ResetWindowsPassword` instance from the injector.